benefits/static/css/admin/styles.css

```diff
@@ -193,6 +193,62 @@ iframe.card-collection {
   display: none;
 }
 
+/* Login Page link styles */
+.login a:not(.btn):not(.card) {
+  color: var(--primary-color);
+  text-decoration: underline;
+  text-decoration-style: solid;
```

Comment:
Change this to `text-decoration: none;`

Reply:
I think I'd recommend changing the `not(.btn)` to `not(.google-btn-label)`

Reply:
`not(.google-btn-label)` doesn't work since that class is on the `<div>` that wraps the `<a>`.

I used `:not(.btn-content)` which is the class applied to the Google SSO link, and that worked [`58148dd`](https://github.com/cal-itp/benefits/pull/2746/commits/58148ddb5ea8f053858b525539f0468b360f226e)